tests/flows/steps/service_crm_steps/test_transfer_with_organization.py

```diff
@@ -71,6 +72,11 @@ def test_create_transfer_request_ticket_with_organization_step_creates_ticket(
     mpt_client_mock = mocker.Mock(spec=MPTClient)
     next_step_mock = mocker.Mock()
 
+    template_response = Response()
```

Comment:
Really strange mock here... If you are testing the step, in that case you should mock the mpt_client function. If you are testing get_template function inside the mpt_client you need to use responses